src/main/java/com/myroslav/cosmickitties/dto/OrderDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.myroslav.cosmickitties.dto;
+
+import lombok.*;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class OrderDTO {
+    private Long id;
```

Comment:
Id shoulnd't be provided in the request, you should split on the request and response dto, and don't alow to create the id from the api, that's db responsibility